src/Microsoft.AspNetCore.Mvc.Core/ApplicationParts/AssemblyPart.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.AspNetCore.Mvc.ApplicationParts
     /// <summary>
     /// An <see cref="ApplicationPart"/> backed by an <see cref="Assembly"/>.
     /// </summary>
-    public class AssemblyPart : ApplicationPart
+    public class AssemblyPart : ApplicationPart, IApplicationPartTypeProvider
```

Comment:
Not for review, part of https://github.com/aspnet/Mvc/pull/4344 but I needed it here